pywbemtools/pywbemcli/_cmd_instance.py

```diff
@@ -739,7 +751,6 @@ def cmd_instance_associators(context, instancename, options):
                 Role=options['role'],
                 ResultClass=options['resultclass'],
                 ResultRole=options['resultrole'],
-                IncludeQualifiers=options['includequalifiers'],
                 IncludeClassOrigin=options['includeclassorigin'],
                 PropertyList=resolve_propertylist(options['propertylist']))
```

Comment:
IncludeQualifiers is missing as an argument.

Reply:
Fixed last night